modules/ROOT/pages/am-installing.adoc

```diff
@@ -72,19 +72,17 @@ When you extract the `am.zip` file, a folder called `am` is created in the locat
 
 == SOCKS5 Proxy Setting
 
-If you need to proxy your monitoring data as it leaves the data center, you can use the command line parameter `p` when you install the Anypoint Monitoring agent. 
+If you need to proxy your monitoring data as it leaves the data center, you can use the command line parameter `-p` when you install the Anypoint Monitoring agent. 
 
 To use the SOCKS5 protocol for monitoring, you must:
 
 . Set up Runtime Manager with a proxy. See xref:runtime-manager::servers-create.adoc[Create a Server in Runtime Manager].
 . To perform the SOCKS5 proxy configuration step, you must have a compatible server deployed to your environment.
-. Install the Anypoint Monitoring agent and use the `p` option. For example, on a Windows server, at the command line, enter: +
-====
-<location-of-am-folder\bin>powershell -file install.ps1 -p socks5://user:password@socks5-server:1080
-====
-
-
+. Install the Anypoint Monitoring agent and use the `-p` option. For example, on a Windows server, at the command line, enter: 
 
+----
+<location-of-am-folder\bin>powershell -file install.ps1 -p socks5://user:password@socks5-server:1080
```

Comment:
This is for windows - would be great if you could show the Linux example I shared on slack